bundles/asdi/asdi-projection-change/resources/css/style.css

```diff
@@ -0,0 +1,86 @@
+.oskari-projection-changer {
+    height:36px;
+    width: 36px;
+    background-image: url("../images/VäliaikainenMaapalloikoni.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-bottom: 10px !important;
+    cursor: pointer;
+}
+.oskari-map-projection {
+    width: 600px;
+    display: flex;
+    display: -ms-flex;
+    flex-wrap: wrap;
+    overflow: hidden;
+    align-items: center;
+    justify-content: center;
+    padding-left: 3%;
+    overflow-x: hidden;
+}
+/*
+ * Card
+ */
+.projection-card {
+    width: 100%;
+    height: 160px;
+    margin: 3px;
+    margin-bottom: 10px;
+    cursor: pointer;
+    flex: 30%;
+    -ms-flex: 30%;
+}
+.projection-info {
+}
+.projection-warning {
+    display: none;
+    height:10px;
+    width: 10px;
+    background-color: green;
+}
+.info-row {
```

Comment:
This is really generic class. I think you should prefix everything in this file with .oskari-projection-changer or something.